ext/storage_sources/s3_store/s3_log_system.cpp

```diff
@@ -33,55 +62,48 @@ S3LogSystem::Log(Aws::Utils::Logging::LogLevel logLevel, const char *tag, const
     va_end(args);
 }
 
+// Writes the log stream to the output stream.
 void
 S3LogSystem::LogStream(
   Aws::Utils::Logging::LogLevel logLevel, const char *tag, const Aws::OStringStream &messageStream)
 {
     LogAwsMessage(tag, messageStream.rdbuf()->str().c_str());
 }
 
+// Directs the output stream to WiredTiger's log streams.
```

Comment:
I am not following the reference to "output stream" in the comments on this file. This function seems to be taking a string and putting it into error-log stream of wiredtiger. There is a WiredTiger log stream here, not sure about the other "output stream".

Reply:
Yes, I will change this to say message instead of output stream.